app/components/shared/card_component.rb

```diff
@@ -0,0 +1,41 @@
+class Shared::CardComponent < GovukComponent::Base
+  include ViewComponent::SlotableV2
+
+  renders_many :action_items, "ActionItemComponent"
+  renders_many :body_items, "ItemComponent"
+  renders_many :header_items, "ItemComponent"
+
+  attr_accessor :html_attributes, :id
+
+  def initialize(id: nil, classes: [], html_attributes: {})
+    super(classes: classes, html_attributes: html_attributes)
+    @id = id
+  end
+
+  class ItemComponent < ViewComponent::Base
```

Comment:
When we've got components this concise, I think it makes sense to use [lambda components ](https://viewcomponent.org/#lambda-slots)